homeassistant/components/backup/manager.py

```diff
@@ -878,6 +887,36 @@ def remove_subscription() -> None:
         self._backup_event_subscriptions.append(on_event)
         return remove_subscription
 
+    def _update_issue_backup_failed(self) -> None:
+        """Update issue registry when a backup fails."""
+        ir.async_create_issue(
+            self.hass,
+            DOMAIN,
+            "automatic_backup_failed",
+            is_fixable=False,
+            is_persistent=True,
+            severity=ir.IssueSeverity.WARNING,
+            translation_key="automatic_backup_failed_create",
+        )
```

Comment:
Should we set the `learn_more_url` to "homeassistant://config/backup" or something?

Reply:
Sounds good.